moon/bulk_test.sh

```diff
@@ -12,7 +12,9 @@ SCORE="13"
 RANGE_START=0
 RANGE_END=49
 
-while getopts "r:t:s:f:l:h" opt; do
+SRCP2_WAIT=5
+
+while getopts "r:t:s:f:l:hw:" opt; do
```

Comment:
isn't this typo? should not be "r:t:s:f:l:h:w"?

Reply:
I don't think so, a colon follows a switch character if the switch has an argument. 'h' for help does not have an argument but 'w' for wait does?

Reply:
I see, thanks for explanation :)